src/main/kotlin/no/skatteetaten/aurora/boober/service/RedeployService.kt

```diff
@@ -32,11 +30,16 @@ class RedeployService(val openShiftClient: OpenShiftClient,
         }
     }
 
-    fun triggerRedeploy(namespace: String, name: String, imageStream: ImageStream?): RedeployResult {
+    fun triggerRedeploy(deploymentConfig: DeploymentConfig, imageStream: ImageStream?): RedeployResult {
+        val namespace = deploymentConfig.metadata.namespace
+        val name = deploymentConfig.metadata.name
+        val imageChangeTriggerTagName = deploymentConfig.findImageChangeTriggerTagName()
+                ?: throw IllegalArgumentException("No imageChangeTriggerName found")
```

Comment:
Tror det var jeg som sa at vi måtte kaste en exception her, men det blir feil for applikasjoner som blir deployet med template. Hvis den ikke har en imagechangetrigger så skal den gjøre en redeploy og ikke en imagestreamimport.

Reply:
Er det en DeployRequest vi skal gjøre i det tilfellet?

Reply:
Hvis du ser en av mine kommentarer lengre ned, @jarlehansen , så kan denne sjekken potensielt legges inn i else-blokken.